src/test/java/uk/ac/ebi/embl/template/TemplateEntryProcessorTest.java

```diff
@@ -0,0 +1,170 @@
+package uk.ac.ebi.embl.template;
+
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.http.*;
+import org.springframework.web.client.RestTemplate;
+import uk.ac.ebi.embl.api.entry.feature.SourceFeature;
+import uk.ac.ebi.embl.api.service.SequenceToolsServices;
+import uk.ac.ebi.embl.api.service.WebinSampleRetrievalService;
+import uk.ac.ebi.embl.api.validation.Origin;
+import uk.ac.ebi.embl.api.validation.ValidationMessage;
+import uk.ac.ebi.embl.api.validation.submission.SubmissionOptions;
+import uk.ac.ebi.ena.webin.cli.validator.reference.Sample;
+
+import java.io.File;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+public class TemplateEntryProcessorTest {
+
+    private final static String AUTH_JSON="{\"authRealms\":[\"ENA\"],\"password\":\"sausages\",\"username\":\"Webin-256\"}";
+    private final static String TEST_AUTH_URL="https://wwwdev.ebi.ac.uk/ena/submit/webin/auth/token";
+    private final static File templateFile = Paths.get(System.getProperty("user.dir") + "/src/main/resources/templates/ERT000002.xml").toFile();
+    private final static String MOL_TYPE = "/mol_type";
+    private String token="";
+    private TemplateEntryProcessor templateEntryProcessor;
+    private TemplateVariables templateVariables;
+    
+    @Before
+    public void setUp() throws Exception {
+        templateEntryProcessor = getTemplateEntryProcessor();
+        SequenceToolsServices.init(new WebinSampleRetrievalService(getAuthToken(),true));
+    }
+    
+    @Test
+    public void  testEntryProcess_ERT000002() throws Exception {
+
+        TemplateInfo templateInfo_ERT000002 = new TemplateLoader().loadTemplateFromFile(templateFile);
+        String MOL_TYPE_ERT000002 = getMolTypeFromTemplateForTest(templateInfo_ERT000002);
```

Comment:
Would be better to use the variable name ```molType_ERT000002``` not to confuse it with a static final constant.